### PR TITLE
pytorch: add softmax scratch

### PR DESCRIPTION
**Pull request by AnirudhDagar**

### CODE REVIEW

File: chapter_linear-networks/softmax-regression-scratch.md [hunk 1]
```diff
@@ -9,17 +9,33 @@ As with linear regression, after doing things by hand
 we will breeze through an implementation in Gluon for comparison.
 To begin, let us import the familiar packages.
 
-```{.python .input  n=1}
+```{.python .input}
+import sys
+sys.path.insert(0, "../")
```
Comment (mli) [inline chapter_linear-networks/softmax-regression-scratch.md:14]: We don’t need it as we assume users have already install d2l
